tools/importer-msgraph-metadata/pipeline/task_template_operations.go

```diff
@@ -0,0 +1,277 @@
+package pipeline
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func (p pipelineTask) templateOperationsForService(resources Resources) error {
+	operations := make(map[string]string)
+
+	// First build all the methods
+	for _, resource := range resources {
+		if resource.Category == "" {
+			continue // TODO do something about orphaned resources
+		}
+
+		for _, operation := range resource.Operations {
+			// Determine request model
+			var requestModel string
+			if operation.RequestModel != nil {
+				requestModel = *operation.RequestModel
+			}
+
+			// Determine response model and return values
+			var responseModel string
+			if operation.Type != OperationTypeDelete {
+				responseModel = operation.Responses.FindModelName()
+				if responseModel == "" {
+					if operation.ResourceId != nil && len(operation.ResourceId.Segments) > 0 && operation.ResourceId.Segments[len(operation.ResourceId.Segments)-1].Value == "$ref" {
+						responseModel = "DirectoryObject"
+					}
+				}
+			}
+
+			statuses := make([]string, 0)
+			for _, response := range operation.Responses {
+				if response.Status >= 200 && response.Status < 400 {
```

Comment:
what's the use-case fo supporting `3xx` status codes?

Reply:
I don't believe there is one, removed!